### DISCUSSION TITLE
How would I have different blocking schedules for different web sites?

### DISCUSSION THREAD

**Discussion by tinker123456** [AXeL-dev/distract-me-not#61] [General]
How would I have different blocking schedules for different web sites?

For example, lets say I want to block twitter.com and facebook.com from 7am - 7PM, BUT, I want to be able to view reddit.com from 9am-9:30am,  noon-12:30pm, and 4:00PM - 4:30PM?

Love the extension so far.  It has helped me from impulsively clicking  on shortcut links.

**Comment by AXeL-dev** [comment]
Hum, that looks a nice idea, still for now it's not yet possible, I guess that one of the solutions could be to implement multiple blacklists/whitelists #15, then on each schedule time, provide/choose a specific blacklist and/or whitelist to use on that schedule.

Can you please open a feature request for that on the [issues section](https://github.com/AXeL-dev/distract-me-not/issues)?
